sources/osg/Shape.js

```diff
@@ -330,9 +330,9 @@ define( [
         var vertexes = new MACROUTILS.Float32Array( [ -1.0, -1.0, -1.0, 4.0, 4.0, -1.0 ] );
 
         var indexes = new MACROUTILS.Uint16Array( 3 );
-        indexes[ 0 ] = 0;
+        indexes[ 0 ] = 2;
```

Comment:
before the change:
- no face culling on composer (waste of gpu resource)
- FS triangle NEEDED culling disabled otherwise it showed nothing at all.
( backface culling woudln't work (not intuitive at all)) 

after the change:
- FS triangle order is now following default order ( one allowing backface culling)
- add backface culliing (no more waste of gpu resource, faster path )